conda_smithy/templates/circle.yml.tmpl

```diff
@@ -17,8 +19,16 @@ dependencies:
   override:
     - docker pull condaforge/linux-anvil
 
+{% endif -%}
+{% endblock -%}
 test:
   override:
+{%- block env_test %}
+{%- if matrix %}
     # Run, test and (if we have a BINSTAR_TOKEN) upload the distributions.
     - ./ci_support/run_docker_build.sh
-{# Keep the terminal newline. #}
+{% else %}
+    # In case the skip fails for some reason, do a fast finish.
```

Comment:
> What does it mean that it is "skipped"? The automatic rerender should disable the circleci build?

Yes, it should. Nearly always it does as described in this PR.

> The final file basically only has this comment...

Not totally true. The file has a [regex statement]( https://github.com/conda-forge/anypytools-feedstock/blob/16a019625aef555ded1641ee6b12e12df13c685b/circle.yml#L1-L4 ) that says not to build on any branch. As a result, all builds should be skipped.